committee-steering/README.md

```diff
@@ -0,0 +1,50 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Steering Committee
+
+The Kubernetes Steering Committee is the governing body of the Kubernetes project, providing decision-making and oversight pertaining to the Kubernetes project bylaws, sub-organizations, and financial planning. The Steering Committee also defines the project values and structure.
+
+The [charter](https://git.k8s.io/steering/charter.md) defines the scope and governance of the Steering Committee.
+
+## Meetings
+* Public Steering Committee Meeting: [Wednesdays at 13:00 PT (Pacific Time)](https://bit.ly/k8s-steering-wd) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=13:00&tz=PT%20%28Pacific%20Time%29).
+  * [Meeting recordings](https://www.youtube.com/watch?v=YAzgJRQxsdc&list=PL69nYSiGNLP1yP1B_nd9-drjoxp0Q14qM).
+
+## Members
+
+* Brian Grant (**[@bgrant0607](https://github.com/bgrant0607)**), Google
+* Brendan Burns (**[@brendanburns](https://github.com/brendanburns)**), Microsoft
+* Derek Carr (**[@derekwaynecarr](https://github.com/derekwaynecarr)**), Red Hat
+* Davanum Srinivas (**[@dims](https://github.com/dims)**), Huawei
+* Joe Beda (**[@jbeda](https://github.com/jbeda)**), VMware
```

Comment:
@jbeda and @timothysc should your affiliation be Heptio or VMware? It says Heptio in the steering repo: https://github.com/kubernetes/steering#members :)